.github/workflows/continuous-integration.yml

```diff
@@ -24,6 +24,8 @@ jobs:
         include:
           - php-version: "8.0"
             dbal-version: "2.13"
+          - php-version: "8.0"
+            dbal-version: "3.2@dev"
```

Comment:
What should happen to this when 3.2.0 is released? Should we change it to whatever the latest 3.1.x is then?

Reply:
We heavily depend on DBAL. I think we should always have a CI job that checks against a dev version of DBAL. When 3.2.0 is released, this job will test against 3.3-dev automatically and I think that's fine.

Reply:
Is there a syntax we could use to avoid this `2` being hardcoded here?